src/utils/storage/index.ts

```diff
@@ -120,6 +122,9 @@ export interface IStorageUtils {
   anonymousToken: Storage;
   lastPromptNotification: Storage;
   lastSelectedMenu: Storage;
+  blockingStatus: Storage;
+  blockedStatus: Storage;
+  incognitoCreatePostFirstTime: Storage;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `incognitoCreatePostFirstTime` property has been added to the `IStorageUtils` interface, which ensures type safety when accessing this storage key. This is a good practice as it helps prevent runtime errors due to incorrect types.
```diff
+   blockingStatus: Storage;
+   blockedStatus: Storage;
+   incognitoCreatePostFirstTime: Storage;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->